addons/point_of_sale/models/pos_session.py

```diff
@@ -188,6 +188,8 @@ def create(self, values):
                 journals = Journal.with_context(ctx).search([('type', '=', 'cash')])
                 if not journals:
                     journals = Journal.with_context(ctx).search([('journal_user', '=', True)])
+            if not journals:
+                raise ValidationError(_("No payment method configured! \nEither Chart of Account not installed or no payment method configured for this POS"))
```

Comment:
"No payment method configured! \nEither no Chart of Account is installed or no payment method is configured for this POS."

Note that you should also export the translation when you add new translatable terms.